core/source/Provider/Mutation.mint

```diff
@@ -5,7 +5,7 @@ record Provider.Mutation.Subscription {
 }
 
 /*
-A provider to provid events when the DOM strucutre of the given
+A provider to provid events when the DOM structure of the given
```

Comment:
`provid` here is off as well but I'm unable to discover the original meaning of this xD

Reply:
It supposed to be `provide` (missed `e` on the end)

Reply:
Uhmm, so it's really `A provider to provide [...]`? 😅 That was too tricky for me :P

Reply:
Yep :joy: